openedx/adg/lms/bootcamps/tests/test_helpers.py

```diff
@@ -0,0 +1,53 @@
+"""
+Tests for all the code in templatetags/helpers.py Bootcamps application.
```

Comment:
a little redundant, maybe change it to:

"Tests for template tags' helper methods"